compiler+runtime/include/cpp/jank/runtime/obj/persistent_array_map.hpp

```diff
@@ -3,6 +3,7 @@
 #include <jank/runtime/object.hpp>
 #include <jank/runtime/detail/native_persistent_array_map.hpp>
 #include <jank/runtime/obj/persistent_array_map_sequence.hpp>
+#include <jank/runtime/obj/persistent_hash_map.hpp>
```

Comment:
Please forward declare this instead, as we do with `persistent_array_map_ptr` below.